lib/wallaby/phantom/driver.ex

```diff
@@ -37,7 +37,8 @@ defmodule Wallaby.Phantom.Driver do
       session_url: base_url <> "session/#{id}",
       url: base_url <> "session/#{id}",
       id: id,
-      server: server
+      server: server,
+      driver: Wallaby.Phantom
```

Comment:
`Wallaby.Phantom.Driver` is going to be an internal module that will eventually go away and have it's logic split between to `Wallaby.Phantom` and `Wallaby.Phantom.WebdriverClient`. Check out [my comment below](https://github.com/keathley/wallaby/pull/201#discussion_r112809389) for an example of how I'm doing it with my selenium driver.